dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportControllerByIdTest.java

```diff
@@ -687,6 +695,69 @@ private EventDataValue dataValue(DataElement de, String value) {
     return dv;
   }
 
+  @Test
+  void getEventChangeLog() {
+    Event event = event(enrollment(trackedEntity()));
+    event.getEventDataValues().add(dv);
+    manager.update(event);
+
+    String json =
+        """
+      {
+        "events": [
+          {
+            "event": "%s",
+            "status": "COMPLETED",
+            "program": "%s",
+            "programStage": "%s",
+            "enrollment": "%s",
+            "trackedEntity": "%s",
+            "orgUnit": "%s",
+            "occurredAt": "2023-01-10",
+            "scheduledAt": "2023-01-10",
+            "storedBy": "tracker",
+            "followUp": false,
+            "deleted": false,
+            "createdAt": "2018-01-20T10:44:03.222",
+            "createdAtClient": "2017-01-20T10:44:03.222",
+            "updatedAt": "2018-01-20T10:44:33.777",
+            "completedBy": "tracker",
+            "completedAt": "2023-01-20",
+            "notes": [],
+            "followup": false,
+            "geometry": null,
+            "dataValues": [
+              {
+                "dataElement": "%s",
+                "value": "new value"
+              }
+            ]
+          }
+        ]
+      }}
+        """
+            .formatted(
+                event.getUid(),
+                program.getUid(),
+                programStage.getUid(),
+                event.getEnrollment().getUid(),
+                event.getEnrollment().getTrackedEntity().getUid(),
+                event.getOrganisationUnit().getUid(),
+                event.getEventDataValues().iterator().next().getDataElement());
+
+    JsonWebMessage msg =
+        POST("/tracker?async=false&importStrategy=UPDATE", json)
+            .content(HttpStatus.OK)
+            .as(JsonWebMessage.class);
+
+    JsonEvent changeLogJson =
+        GET("/tracker/events/{id}/changelog", event.getUid())
+            .content(HttpStatus.OK)
+            .as(JsonEvent.class);
```

Comment:
That is not intentional right? We are not returning an event here 🤔

Reply:
yeah, don't pay too much attention to this test class... yet